tests/contrib/pytest/test_pytest.py

```diff
@@ -515,14 +519,11 @@ def test_dd_origin_tag_propagated_to_every_span(self):
         py_file = self.testdir.makepyfile(
             """
             import pytest
-            import ddtrace
-            from ddtrace import Pin
 
-            def test_service(ddspan, pytestconfig):
-                tracer = Pin.get_from(pytestconfig).tracer
-                with tracer.trace("SPAN2") as span2:
-                    with tracer.trace("SPAN3") as span3:
-                        with tracer.trace("SPAN4") as span4:
+            def test_service(ddtracer):
```

Comment:
is this how the tracer will be available now? Or this is in addition to being able to do `import ddtrace`

Reply:
Yes. It is the recommended way to make sure that only the CI configured tracer is used. Normally it is the same one as `ddtrace.tracer` but it's an extra precaution in case we decide to remove side-effects on `ddtrace` import.